doc/sphinxext/ipython_sphinxext/ipython_directive.py

```diff
@@ -127,7 +127,11 @@
 from sphinx.util.compat import Directive
 
 # Our own
-from IPython import Config, InteractiveShell
+try:
+    from traitlets.config import Config
+except ImportError:
```

Comment:
I don't know. Possibly they forgot to put a shim for Config (but this code is normally included in ipython itself, so then it doesn't matter. But is seems that Config is rather public object)